tests/cucumber/features/organizations.feature

```diff
@@ -3,6 +3,7 @@ Feature: Organizations
   Background:
     Given there is a test user in the database
 
+  @organizations
```

Comment:
Did you add this tag so that you could run all of the organization tests while developing?  I'm just curious, I've been running one test at a time with the @dev tag.